testsuite/features/step_definitions/common_steps.rb

```diff
@@ -957,8 +957,8 @@ def server_secret
   end
 end
 
-Then(/^I should see a list item with text "([^"]*)" and bullet with "([^"]*)" icon$/) do |text, class_name|
-  item_xpath = "//ul/li[text()='#{text}']/i[contains(@class, 'text-#{class_name}')]"
+Then(/^I should see a list item with text "([^"]*)" and bullet with style "([^"]*)"$/) do |text, class_name|
+  item_xpath = "//ul/li[text()='#{text}']/i[contains(@class, '#{class_name}')]"
```

Comment:
Just keep an eye if that will break 3.2, in case we port.

Reply:
Should work in the same way in 3.2.